documentation/asciidoc/computers/raspberry-pi/boot-msd.adoc

```diff
@@ -24,9 +24,9 @@ Please see the xref:compute-module.adoc#flashing-the-compute-module-emmc[Flashin
 
 The Raspberry Pi 3B+ supports USB mass storage boot out of the box.
 
-=== Raspberry Pi 2B, 3A+, 3B, CM 3, 3+, Zero 2
+=== Raspberry Pi 2B, 3A+, 3B, CM 3, 3+, Zero 2 W
 
-On the Raspberry Pi 2B v1.2, 3A+, 3B, Zero 2, and Compute Module 3, 3+ you must first enable xref:raspberry-pi.adoc#usb-host-boot-mode[USB host boot mode]. This is to allow USB mass storage boot, and xref:raspberry-pi.adoc#network-booting[network boot]. Note that network boot is not supported on the Raspberry Pi 3A+.
+On the Raspberry Pi 2B v1.2, 3A+, 3B, Zero 2 W, and Compute Module 3, 3+ you must first enable xref:raspberry-pi.adoc#usb-host-boot-mode[USB host boot mode]. This is to allow USB mass storage boot, and xref:raspberry-pi.adoc#network-booting[network boot]. Note that network boot is not supported on the Raspberry Pi 3A+.
```

Comment:
@pelwell Presumably network-boot isn't supported on the Zero 2 W either? (i.e. it's only supported on Raspberry Pis with a built-in Ethernet port?)

Reply:
The CM3 doesn't have on-board Ethernet either, but like the Zero 2 W (and, in a sense, the 3B) it will net boot if paired with a LAN9514/SMSC95XX USB-attached Ethernet chip.

Reply:
Ahh, thanks for the info! Does that mean that the "Note that network boot is not supported on the Raspberry Pi 3A+." part is in fact incorrect?

Reply:
Do you know why 3A+ is excluded, @ghollingworth?

Reply:
I think just because of the lack of Ethernet...

Reply:
Drop the Zero 2 and 3A+ then - it's safer.